Formula/rpki-client.rb

```diff
@@ -0,0 +1,49 @@
+class RpkiClient < Formula
+  desc "OpenBSD portable rpki-client"
+  homepage "https://github.com/rpki-client/rpki-client-portable"
+  url "https://github.com/rpki-client/rpki-client-portable/archive/refs/tags/7.4.tar.gz"
+  sha256 "cacb56107b29baef67c0117bc53d726c95684ad27a6affe2a1527470c67f5e86"
+  license "ISC"
+
+  depends_on arch: :intel
+  depends_on "autoconf"
+  depends_on "automake"
+  depends_on "libressl"
+  depends_on "libtool"
+  depends_on macos: :big_sur
+  depends_on "pkg-config"
+
+  def install
+    ENV.deparallelize
+
+    # git fetch the OpenBSD head codestate, and apply the patches to convert it to portable
+    system "./autogen.sh"
+
+    # --localstatedir wires things into the installed /var/{db,cache}/rpki-client paths, not the symlink farm
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}",
+                          "--sysconfdir=#{HOMEBREW_PREFIX}/etc/",
+                          "--localstatedir=#{HOMEBREW_PREFIX}/var"
+    system "make", "install"
+  end
+
+  def post_install
+    # make the var/db,cache/rpki-client dirs
+    (var/"db/rpki-client").mkpath
+    (var/"cache/rpki-client").mkpath
+  end
+
+  def caveats
+    <<~EOS
+      If you accept the ARIN Relying Party Agreement (RPA), and want to install the arin.tal:
+
+      curl -o "#{etc}/rpki/arin.tal" https://www.arin.net/resources/manage/rpki/arin.tal
+    EOS
+  end
+
+  test do
+    system "#{sbin}/rpki-client", "-V"
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
I was worried that the system is proved correct by fetching external state and writing into /var/cache/rpki-client/ and /var/db/rpki-client/ paths. I can probably get a testbed CA or use the APNIC AS0 CA as a single-instance test to confirm it can work with this, but it means the test will "dirty" the runtime state, and/or create traffic on the network. So, if you wanted a "silent" install, you'd be visible to the world if you ran test in this situation.

Reply:
I deployed a test which checks for the timeout() exit message.